packages/jest-changed-files/src/git.ts

```diff
@@ -30,7 +28,7 @@ const adapter: SCMAdapter = {
     cwd: string,
     options?: Options,
   ): Promise<Array<Path>> => {
-    const changedSince: ?string =
+    const changedSince: string | null | undefined =
```

Comment:
I know this is the correct flow->TS conversion, but is it correct? Can it be both `null` and `undefined`? If not, pleas narrow it down to only actual possibilities

Reply:
This can only be `null`, so i've updated the type to `string | null`